### ISSUE TITLE
training

### ISSUE THREAD

**Issue by tensorflowt** [ppplinday/tensorflow-vgg16-train-and-test#3]
Hello! I would like to ask if your training code can be used to train ILSVRC-2012 dataset？ Thank you！

**Comment by ppplinday** [comment]
Hi, I think this code cannot train ILSVRC-2012 dataset cause this is for study report. And also i will recommend you to see my another repositories: [https://github.com/ppplinday/TFrecord-Lenet-Vgg19-tensorflow](url), I think this one is better.

**Comment by kusiwu** [comment]
hi,
I am training your examples but the accuracy and losses are same in each epochs.
I am trying to train cifar-10 data set  and test it with my any 32*32 picture. Any links or examples for this?
i used [this](https://github.com/mjiansun/cifar10-vgg16) but, there is no testing example. It produced .h5 file and i cant test it with a random image to get classification result. Any help would be great

**Comment by ppplinday** [comment]
Hi,  since this is for study report and  if you want to train i will recommend you to see my another repositories: https://github.com/ppplinday/TFrecord-Lenet-Vgg19-tensorflow, I think this one is better and it can help you to train cifar-10 data set! Hope this can help!

**Comment by abaooaba** [comment]
Hi,excuse me,I am training your examples but the accuracy and losses are same in each epochs.Do you know what caused it?Any help would be great！Thank you！

**Comment by kusiwu** [comment]
> Hi,excuse me,I am training your examples but the accuracy and losses are same in each epochs.Do you know what caused it?Any help would be great！Thank you！

Please visit my repository "cifar-10 vgg16"

**Comment by ppplinday** [comment]
> Hi,excuse me,I am training your examples but the accuracy and losses are same in each epochs.Do you know what caused it?Any help would be great！Thank you！

Please visit my repository also :[ https://github.com/ppplinday/TFrecord-Lenet-Vgg19-tensorflow](url), thank you so much!

**Comment by wy9933** [comment]
have you solved this issue?I also confused about it @abaooaba
